### ISSUE TITLE
[Audiences] Give some love to showContact.php

### ISSUE THREAD

**Issue by daver78** [coa0329/BhaangWeb#75]
Area needs a lot of tender touching.

**Comment by daver78** [comment]
Before: 

![image](https://user-images.githubusercontent.com/7561473/30938432-d94a9d46-a38e-11e7-8f8a-51e6b6accbab.png)

**Comment by daver78** [comment]
After:

![image](https://user-images.githubusercontent.com/7561473/30941564-f181595c-a39a-11e7-8adf-518544f693fa.png)

**Comment by daver78** [comment]
@paagla123 OK, I gave this some minimal love so it is presentable @ MarTech.  I eliminated the `target=_new` from the previous page.  Are you OK with that?

Regarding drill down to the profile view, I think we should consider how to incorporate the CRM features you've found useful to CLients (log a call, add a task, etc.) -- in addition to the marketing attributes required to generate leads and sales.

What say you?
